modules/tensor_mechanics/src/postprocessors/MaterialTimeStepPostprocessor.C

```diff
@@ -28,38 +28,103 @@ validParams<MaterialTimeStepPostprocessor>()
   params.addClassDescription("This postprocessor estimates a timestep that reduces the increment "
                              "change in a material property below a given threshold.");
 
+  params.addParam<bool>("use_matl_timestep_limit",
+                        true,
+                        "if true, the time step is limited by the minimum value of the "
+                        "matl_timestep_limit property");
+
+  params.addParam<MaterialPropertyName>("elements_changed_property",
+                                        "name of the property used to evaluate if an"
+                                        "element have changed");
+
+  params.addRangeCheckedParam<int>("elements_changed_number",
+                                   "elements_changed_number > 0",
+                                   "number of elements used to determine if a change of"
+                                   "time step is required");
```

Comment:
Change description to:

Maximum number of elements within which the property named in 'elements_changed_property' is allowed to change by more than 'elements_changed_threshold' before the time step is limited.